packages/lab/src/Flow/Node/Node.tsx

```diff
@@ -144,11 +160,28 @@ export const HvFlowNode = ({
     >
       {(subtitle || actions) && (
         <div className={classes.subtitleContainer}>
```

Comment:
Can we update the padding of the `subtitleContainer` to `padding: theme.space.xs` when `disableInlineEdit` is `true` to remove the unneeded extract left padding?